drivers/sensor/sensirion/sts4x/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_library()
+
+zephyr_library_sources(sts4x.c)
```

Comment:
Looks like github wants newline after the last line in a file. Not sure if this is a Zephyr requirement though.